examples/vector_collection/vector_collection.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from hazelcast import vector
+from hazelcast.core import HazelcastJsonValue
+from hazelcast.vector import Type, Vector
+
+logging.basicConfig(level=logging.DEBUG)
+
+from hazelcast.client import HazelcastClient, Config
+from hazelcast.proxy.vector_collection import Document
+
+
+def main():
+    # Connect to the cluster
+    cfg = Config()
+    cfg.cluster_members = ["localhost:5701"]
+    client = HazelcastClient(cfg)
+
+    vc_name = "my-vector-collection"
+
+    # Create the VectorCollection
+    indexes = [
+        vector.IndexConfig(name="default-vector", metric=vector.Metric.COSINE, dimension=2),
+    ]
+    client.create_vector_collection(vc_name, indexes=indexes)
```

Comment:
interesting that we need to do create() and then get() as opposed to other data structures, do we need a create() just for vector collection?

ps: I didn't see java API if there is, can you link it if there is?

Reply:
Java API is different from the Python API.
There's currently not a decision about which way to go:
https://github.com/hazelcast/hazelcast-mono/blob/064036486c8da3725b2659995598618715f31904/hazelcast-enterprise-vector/src/main/java/com/hazelcast/vector/VectorCollection.java#L101